registry/handlers/helpers.go

```diff
@@ -2,6 +2,7 @@ package handlers
 
 import (
 	"encoding/json"
+	"github.com/docker/distribution/registry/api/errcode"
```

Comment:
oops - i think I added at the end w/o a blank line by mistake and then gofmt alphabetized 'em.  Fixed.